### ISSUE TITLE
Pointmass weights question

### ISSUE THREAD

**Issue by wlbragg** [Juanvvc/c172p-detailed#638]
Is this /fdm/jsbsim/inertia/pointmass-weight-lbs[0] or [1]?
~~~xml
       <pointmass name="Pilot">
            <weight unit="LBS"> 180 </weight>
            <location name="POINTMASS" unit="IN">
                <x> 36 </x>
                <y> -14 </y>
                <z> 24 </z>
            </location>
        </pointmass>
~~~
I'm not sure the question matters as to there being an issue with the pointmass's.
If Pilot is [0] then
~~~xml 
<pointmass name="Left bush wheel 26 inches">
            <weight unit="LBS"> 0 </weight>
            <location name="POINTMASS" unit="IN">
                <x> 58.2 </x>
                <y>  -43 </y>
                <z> -15.5 </z>
            </location>
        </pointmass>
~~~
is correct at [5] as in
~~~xml
switch name="extra-weight-left-26-in">
            <default value="0"/>
            <test logic="AND" value="10">
                bushkit EQ 1
            </test>
            <output>/fdm/jsbsim/inertia/pointmass-weight-lbs[5]</output>
        </switch>
~~~
but then
~~~xml
       <pointmass name="Left ski">
            <weight unit="LBS"> 0 </weight>
            <location name="POINTMASS" unit="IN">
                <x> 58.2 </x>
                <y>  -43 </y>
                <z> -15.5 </z>
            </location>
        </pointmass>
        <pointmass name="Right ski">
            <weight unit="LBS"> 0 </weight>
            <location name="POINTMASS" unit="IN">
                <x> 58.2 </x>
                <y>  43 </y>
                <z> -15.5 </z>
            </location>
        </pointmass>
~~~
are wrong.

@Onox, I think this is really screwed up up. Please take a look and let me know what you think

**Comment by wlbragg** [comment]
This is on the assumption that the [?] position is defined at loading position in the FDM, or is it defined somewhere else I am unaware of. We use [13] for engine when I calculate it at [15].

**Comment by dany93** [comment]
If I remember well... Here is my feeling:

In c172p.xml (FDM) I think I had originally written (to help at reading)
        `
        `

but someone (you ?), when adding the skis, changed it for
        `
        `

My hypothesis is (I'm almost sure) that the number is the position in the FDM.
The issue is that you have inserted the skis "bushkit" in [13] and [14] position in the FDM, thus the engine becomes [15].

Please think well, but this should be the issue.

**Comment by wlbragg** [comment]
Yes, that sounds about right except I'm not sure who or how the engine pointmass got positioned below the skies. I would have thought I would have seen the organization at the time I added the skis, but who knows. I'll take another look at it and see if I can reorganize it.

**Comment by onox** [comment]
So left ski is 13, right ski is 14, engine is 15, and oil is 16?

Then lines 50, 58, and 66 in `Systems/bushkit.xml` should be fixed. I have a PR ready in a second.

**Comment by onox** [comment]
@wlbragg I now see you have already created a PR (if you write something like `Fixes #638` in the PR description then Github creates a link here on this issue page)

I think the `<pointmass>` elements should remain organized (all the bushkit pointmasses were already nicely put together by you) as they used to be, and instead update the indices. Which is what I did in my PR.
